### PR TITLE
add AddAddress endpoint, integration tests

### PR DESCRIPTION
**Pull request by badekk**
## add AddAddress endpoint, integration tests

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?

- [ ] Unit test(s)
- [x] Integration test(s)
- [ ] Tested manually

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings
- [ ] I have added frontend code and checked it for mobile & desktop

### CODE REVIEW

**Comment by mar-pan** [comment]
Task linked: [CU-862jbtmvr Add address](https://app.clickup.com/t/862jbtmvr)

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                StreetNumber = newAddress.StreetNumber,
+                Apartment = newAddress.Apartment,
+                GpsLatitude = newAddress.GpsLatitude,
+                GpsLongitude = newAddress.GpsLongitude
+            };
+
+            return response;
+        }
+
+        private static bool ValidatePayload(CreateOrderAddAddressPayload payload)
```
Comment (h4570) [inline src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:101]: Do wywalenia.
Skoro mamy FluentValidation to go używaj. Sprawdź jak robiona jest walidacja w innych klasach.

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                Status = DbEntityStatus.Active
+            };
+            await _appDbContext.AddAsync(newAddress);
+
+            var newUserAddress = new UserAddress()
+            {
+                User = currentUser,
+                Address = newAddress,
+                IsDeletedByUser = false,
+                UpdateUserId = UserSeeder.ServiceUser.Id,
```
Comment (h4570) [inline src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:77]: Nie. Przeczytaj:
https://app.clickup.com/t/862jdza9j

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            {
+                Country = country.Result,
+                PostalCode = payload.PostalCode,
+                City = payload.City,
+                Street = payload.Street,
+                StreetNumber = payload.StreetNumber,
+                Apartment = payload.Apartment,
+                GpsLatitude = payload.GpsLatitude,
+                GpsLongitude = payload.GpsLongitude,
+                UpdateUserId = UserSeeder.ServiceUser.Id,
```
Comment (h4570) [inline src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:66]: To samo.

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+using Core.Database.Seeders;
+using Core.Exceptions;
+using Core.Services;
+using Core.Setup.Auth0;
+using HashidsNet;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Orders.Actions.CreateOrderActions.CreateOrderAddAddress;
+
+namespace Orders.Actions.CreateOrderActions.CreateOrderAddAddresses
```
Comment (h4570) [inline src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:13]: Czemu Addresses a w akcji masz Address?

**Review by h4570 — CHANGES_REQUESTED** [review]

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs (hunk 1 above)
Comment (badekk) [reply src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:101]: wywalone, poprawione

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs (hunk 4 above)
Comment (badekk) [reply src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:13]: fix po zmianie nazwy

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs (hunk 3 above)
Comment (badekk) [reply src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs]: changed

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs (hunk 2 above)
Comment (badekk) [reply src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs]: changed

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs [hunk 5]
```diff
@@ -19,29 +18,28 @@ public class CreateOrderAddAddressService
         private readonly Hashids _hashids;
         private readonly ICurrentUserService _currentUserService;
         private readonly GetCurrentUserService _getCurrentUserService;
+        private readonly CreateOrderAddAddressPayloadValidator _ceateOrderAddAddressPayloadValidator;
```
Comment (h4570) [inline src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:21]: Literówka ceate

File: src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs [hunk 6]
```diff
@@ -19,29 +18,28 @@ public class CreateOrderAddAddressService
...
 
         public CreateOrderAddAddressService(
             AppDbContext appDbContext,
             ILogger<CreateOrderAddAddressService> logger,
             Hashids hashids,
             ICurrentUserService currentUserService,
-            GetCurrentUserService getCurrentUserService
+            GetCurrentUserService getCurrentUserService,
+            CreateOrderAddAddressPayloadValidator ceateOrderAddAddressPayloadValidator
```
Comment (h4570) [inline src/backend/Order/Actions/CreateOrderActions/CreateOrderAddAddress/CreateOrderAddAddressService.cs:29]: Tu też. Wgl staraj się zamknięcie nawiasu dawać na końcu, a nie w nowej linii. 
Bo VS buguje formatowanie czasami przez to.

**Review by h4570 — APPROVED** [review]
Popraw kosmetykę i merguj

**Pull request merged**